doc/source/getting_started/10min.rst

```diff
@@ -468,6 +468,13 @@ Concatenating pandas objects together with :func:`concat`:
 
    pd.concat(pieces)
 
+.. note::
+   Adding a column to a DataFrame is relatively fast. Adding a row, however,
+   requires a full copy of the data and a construction of a new Index, which
```

Comment:
use double-backticks around DataFrame. Also let's make this shorter / simpler.

first part do something like

```
Adding a column to a DataFrame is relatively fast, However, adding a row requires a copy, and may be expensive.
```

Reply:
@jreback Done. Let me know if any more changes are required.